benchmarks/cp/test_gdn_with_cp.py

```
@@ -0,0 +1,924 @@
+    parser.add_argument("--mean", type=int, default=1024*32)
+    parser.add_argument("--std", type=int, default=0)
+    parser.add_argument("--use-cp2hp", action="store_true")
+    return parser.parse_args()
+
+# torchrun --nproc-per-node=4 test_cp.py
+
+
+def a2a(x, stage=1, group=None, async_op=False):
+    assert stage in [1, 2]
+    x = x.contiguous()
+    rank = dist.get_rank(group)
```

Flake8 (7.3.0)

[error] local variable 'rank' is assigned to but never used

(F841)